### ISSUE TITLE
Installation on Mageia 5 x86_64 - libpng16.so.16 dependency not satisfied

### ISSUE THREAD

**Issue by RafaAguilar** [bohoomil/fontconfig-ultimate#108]
Hi Bohoomil,

First, thank you for your efforts, i was trying to make a local repository of _fontconfig-ultimate_ from [OpenSuse Repo](http://download.opensuse.org/repositories/home:/silencer:/infinality-fedora/Fedora_22/x86_64/), i just downloaded all package an run `genhdlist2  .`, later i added to my repo urls, everything fine 'til this step, when i was trying to install packages, this came up:

```
Can't install:
cairo-infinality-ultimate-1.14.2-12.2.x86_64 (due to unsatisfied dependency libpng16.so.16(PNG16_0)(64bit))
```

However i got that lib installed on `/usr/lib64/libpng16.so.16` but i also has a 32bits version under `/usr/lib64/libpng16.so.16`, my guess is that the link into the compiled sources us the `/usr/lib..` route, and thats why it doesn't find the actual 64bits version.

So, in conclusion, i would like to re-compile or help you to support Mageia 5, how can change that behaviour?.

Regards,
